### PR TITLE
[SHELL32] el-GR.rc: Strip undesired format-string

### PR DESCRIPTION
**Pull request by JoachimHenze**
A trivial change.
I don't know why just el-GR does have this strange format-string placed here.
I checked: No other language does have that.
Very likely an oopsie.

JIRA issue: none

image
image

### CODE REVIEW

**Comment by JoachimHenze** [comment]
Ready for review. I won't add anything more into this PR. It should not collide with the intended WS_TABSTOP-strpping-PR that may be worked on in parallel.

File: dll/win32/shell32/lang/el-GR.rc [hunk 1]
```diff
@@ -396,7 +396,7 @@ FONT 8, "MS Shell Dlg", 0, 0, 0x0
 BEGIN
     ICON IDI_SHELL_OPEN_WITH, -1, 8, 12, 21, 20
     LTEXT "Επιλέξτε το πρόγραμμα με το οποίο θέλετε να ανοίξετε αυτό το αρχείο:", -1, 44, 12, 211, 10
-    LTEXT "Αρχείο:    %s", 14001, 44, 25, 188, 20
+    LTEXT "Αρχείο:    ", 14001, 44, 25, 188, 20
```
Comment (katahiromz) [inline dll/win32/shell32/lang/el-GR.rc:399]: image

**Review by katahiromz — APPROVED** [review]
No problem.

**Review by binarymaster — APPROVED** [review]

**Review by binarymaster — COMMENTED** [review]
It's addendum to 4ff1aa5fac0 (r30784) and 4af7888b29d.

File: dll/win32/shell32/lang/el-GR.rc (hunk 1 above)
Comment (binarymaster) [reply dll/win32/shell32/lang/el-GR.rc:399]: It's addendum to 4ff1aa5fac0 (r30784) where the initial dialog update were not applied to el-GR and 4af7888b29d (0.4.14-dev-1492-g4af7888) where the update was partially applied (AUTOCHECKBOX).

**Pull request merged**